### PR TITLE
round of readme tweaks

### PR DESCRIPTION
**Pull request by SethTisue**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -5,9 +5,19 @@
 [<img src="https://img.shields.io/maven-central/v/org.scala-lang.modules/scala-swing_2.12.svg?label=latest%20release%20for%202.12"/>](http://search.maven.org/#search%7Cga%7C1%7Cg%3Aorg.scala-lang.modules%20a%3Ascala-swing_2.12)
 [![Stories in Ready](https://badge.waffle.io/scala/scala-swing.svg?label=ready&title=Ready)](http://waffle.io/scala/scala-swing)
```
Comment (Philippus) [inline README.md]: Let's remove this link, as Waffle is closing its doors: https://waffle.io/closing-its-doors

Comment (SethTisue) [reply README.md]: done in d4edd9ba0427ad6d621f3c123032948c8ba0b0fa

File: README.md [hunk 2]
```diff
@@ -5,9 +5,19 @@
...
-This is now community maintained by @benhutchison & @Sciss. If you are interested in helping then contact them or @adriaanm.
+This is now community maintained by [@Sciss](https://github.com/Sciss) and [@benhutchison](https://github.com/benhutchison). If you are interested in helping then contact them or [@adriaanm](https://github.com/adriaanm).
 
-This is a UI library that will wrap most of Java Swing for Scala in a straightforward manner. 
+## Adding an sbt dependency
+
+To use scala-swing from sbt, add this to your `build.sbt`:
+
+```
+libraryDependencies += "org.scala-lang.modules" %% "scala-parser-combinators" % "2.1.1"
```
Comment (benhutchison) [inline README.md]: scala-parser-combinators isnt right...

Comment (Philippus) [reply README.md]: fixed in #103

Comment (SethTisue) [reply README.md]: 🤭

**Pull request merged**